cves/kernel/CVE-2019-19069.yml

```diff
@@ -147,10 +149,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: 2019-08-29
```

Comment:
I don't think the answer should be a date for how it was discovered but an explanation of who discovered the vulnerability and how they discovered it